contribute.mdx

```diff
@@ -44,7 +44,7 @@ Get paid for the content you write that we publish. You can get up to €200 dis
     <Message type="important">
       Make sure you read our [contribution guidelines](https://github.com/scaleway/docs-content/blob/854cc72797d4103d0475f35f977f32339d6e94f5/docs/CONTRIBUTING.md) and [code of conduct](https://github.com/scaleway/docs-content/blob/854cc72797d4103d0475f35f977f32339d6e94f5/docs/CODE_OF_CONDUCT.md) before submitting your content. 
     </Message>
-4. Your content is reviewed and published to the Scaleway Documentation website, and we credit your Scaleway account with your reward in the month following the publication! For this purpose, do not forget to add your [Organization ID](/console/my-project/how-to/join-an-organization/#how-to-retrieve-your-organization-id) when writing your documentation request issue in step 2.
+4. Your content is reviewed and published to the Scaleway Documentation website, and we credit your Scaleway account with your reward in the month following the publication! For this purpose, do not forget to add your [Organization ID](/identity-and-access-management/iam/how-to/accept-invitation-to-orga) when writing your documentation request issue in step 2.
```

Comment:
Are you sure this is the link you want to use, and not my-account/how-to/create-an-account/ ?